jwst/stpipe/integration.py

```diff
@@ -0,0 +1,88 @@
+"""
+Entry point implementations.
+"""
+
+
+def get_steps():
```

Comment:
Anything we do with a class decorator or a metaclass would require that the classes all be imported, which introduces a 2-3 second lag on my laptop.  I'd like to avoid that since the CLI runs each command in a new Python environment and would be pretty sluggish.